lms/djangoapps/grades/tests/test_scores.py

```diff
@@ -102,7 +102,7 @@ def _create_submissions_scores(self, submission_value):
 
     def _create_csm_scores(self, csm_value):
         """
-        Creates a stub result from courseware student module for the given values.
+        Creates a stub result from lms.djangoapps.courseware student module for the given values.
```

Comment:
Nit: you probably didn't intend to change this comment (although I guess it still reads somewhat sanely).